app/components/Wallet/SuiWallet/useSuiBalance.tsx

```diff
@@ -0,0 +1,24 @@
+import { useSuiClient, useCurrentAccount } from "@mysten/dapp-kit";
+import { CoinBalance } from "@mysten/sui/client";
+import { useEffect, useState } from "react";
+
+// unfortunately, SUI doesn't have their own hook to get the balance at the moment
+// useSuiBalance fetches the SUI balance in mist
```

Comment:
then lets use it and just change the unit by dividing by `MIST_PER_SUI` constant

Reply:
yes, we have method for this in util